doc/changelog.rst

```diff
@@ -15,6 +15,7 @@ Bugfixes:
 Maintenance:
 
 - Make the install process PEP517 compliant PR #97
+- Drop support for Python 3.6 and 3.7 PR #100
```

Comment:
This probably requires a minor version bump, so 0.14.0 instead of 0.13.1. It would be great if the current support is also summarised in the README.

Reply:
Addressed in 7b042323